digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala

```diff
@@ -288,6 +293,50 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
     RoadLinkDAO.LinkTypeDao.getAllLinkType(linkIds).groupBy(_._1)
   }
 
+  def getAllPrivateRoadAssociationNames(newTransaction: Boolean = true): Seq[String] = {
+    if(newTransaction) {
+      withDynSession {
+        LinkAttributesDao.getAllExistingDistinctValues(privateRoadAssociationPublicId)
+      }
+    } else
+      LinkAttributesDao.getAllExistingDistinctValues(privateRoadAssociationPublicId)
+  }
+
+  def getPrivateRoadsByAssociationName(roadAssociationName: String, newTransaction: Boolean = true): Seq[(Long, String, String)] = {
+    val linkIds = getValuesByRoadAssociationName(roadAssociationName, privateRoadAssociationPublicId, newTransaction).map(_._2)
```

Comment:
these transaction-conditions are true by default, can be removed

Reply:
I did this because I wrote two tests where I wanted to test my methods, and i didn't want to open nested transactions. If everybody agrees i'll leave them like this.